package-lock.json

```diff
@@ -1,6099 +1,8 @@
 {
   "name": "@stoplight/json-schema-merge-allof",
   "version": "0.7.7",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
let's revert this change.
We should be using the new lockfile format. You probably have an outdated version of `npm`

Reply:
Yikes!  Thank you for catching that.  I was indeed using an outdated NPM.

@P0lip , should I revert _all_ changes to this file, or just the `lockfileVersion` change?

Reply:
Nevermind, you fixed it for me!  Thanks!

Reply:
No worries!
@EdVinyard I'm going to release a new version, 0.7.8.